### PR TITLE
index.html: Adds missing speaker links

### LINKED ISSUE #76: Add missing speaker links
Add missing fb links of:
- [x] Michael Christen: https://www.facebook.com/michael.christen
- [x] Carsten Haitzler: https://www.facebook.com/carsten.haitzler
- [x] Matthew Treinish: https://www.facebook.com/treinish
- [x] Ben Sadeghi: https://www.facebook.com/profile.php?id=100007734345812
- [x] Anuvrat Rao: https://www.facebook.com/anuvratrao
- [x] Yohei Yasukawa: https://www.facebook.com/yasulab
- [x] Harshit Dwivedi: https://www.facebook.com/daggerdwivedi
- [x] Saptak Sengupta: https://www.facebook.com/s4pt4ks

### PR DESCRIPTION
**Pull request by abishekvashok**
Adds missing facebook links of 8 speaker.
Fixes #76 
Live preview: https://abishekvashok.github.io/2017.fossasia.org/

### CODE REVIEW

**Comment by louisgag** [comment]
Hi Abishek,
just  curious, what's the purpose of the new .directory file?

File: .directory [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```
Comment (akshatnitd) [inline .directory]: Please remove this file from the commit!

**Review by akshatnitd — CHANGES_REQUESTED** [review]

File: .directory (hunk 1 above)
Comment (abishekvashok) [reply .directory]: My kde installation must have sucked me up.

**Comment by abishekvashok** [comment]
@akshatnitd @louisgag done.

> Hi Abishek,
just curious, what's the purpose of the new .directory file?

It was created automatically by my File explorer in kde, Dolphin. Trying to disable it now. Although I have removed it from the PR.

**Comment by abishekvashok** [comment]
Ping @fossasia/website-admins

**Review by dilpreetsio — APPROVED** [review]

**Pull request merged**